static-container-registry.py

```diff
@@ -79,13 +79,15 @@ def find_images(root):
                 LOGGER.info('Invalid schemaVersion in %s', manifest)
                 continue
 
-            if data.get('mediaType') != \
-                    'application/vnd.docker.distribution.manifest.v2+json':
+            mediaType = data.get('mediaType')
+            if not mediaType in [
+                    'application/vnd.docker.distribution.manifest.v2+json',
+                    'application/vnd.oci.image.manifest.v1+json']:
                 LOGGER.info('Invalid mediaType in %s', manifest)
```

Comment:
Just thought about this:
- print the variable `mediaType` here so when someone tries to load it if does not complies it know which media type is wrong.